cli.js

```diff
@@ -31,6 +34,13 @@ function run() {
       process.cwd(),
     ], {stdio: 'inherit'});
     break;
+  case 'link':
+    if (!args[1]) {
+      console.error('Usage: react-native link <path>');
```

Comment:
from the description it's not clear how to use it. I think the most logical use case is something like `react-native link react-native-video`